src/main/java/com/devaneios/turmadeelite/controllers/StudentController.java

```diff
@@ -128,4 +129,17 @@
                 .collect(Collectors.toList());
     }
 
+    @Operation(summary = "Listar os alunos ativos e inativos")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Lista de alunos ativos e inativos retornada com sucesso"
+            )
+    })
+    @IsManager
+    @GetMapping("activeInactive")

```

Comment:
Nomeação dos endpoints normalmente é feito em kebab-case. Neste projeto está sendo feito em kebab-case (se algum estiver diferente, já alterar por favor.
https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841